### DISCUSSION TITLE
Add Sentinel plugin support

### DISCUSSION THREAD

**Discussion by Seeeby** [BG-Software-LLC/SuperiorSkyblock2#1698] [Approved Feature Requests]
Hey, is it possible that SSB supports the "Sentinel" plugin? When the NPC tries to attack, it does no damage to the player and it comes from SSB, I tested on a server with only Sentinel, Citizens and SSB 

Thanks !

**Comment by OmerBenGera** [comment]
Make sure you have damage enabled for visitors and island members (look for these options in the config file)

**Reply by Seeeby** [reply]
I tested by enabled the damage and disabled it but still the same problem
